api/api/spec/spec.yaml

```
@@ -310,30 +310,38 @@ x-rbac-catalog:
         actions: ['decoders:delete']
         resources: ['decoder:file:local_decoder.xml']
         effect: "allow"
-    'lists:read':
-      description: "Read CDB lists files"
+    'kvdbs:read':
```

Again, to be consistent with the rest of RBAC policies we should use singular nouns

```suggestion
    'kvdb:read':
```

---

The 's' has been kept for now because the previous '`lists`' endpoint was in plural, and the new endpoints, for example '`decoders`', also come with 's': https://github.com/wazuh/wazuh/blob/a8d2cbb065234de9fe11ac50745bf8eb91f58556/api/api/spec/spec.yaml#L287